### PR TITLE
fix: add page self-contained; add test in the playground

### LINKED ISSUE #273: add self-contained styles for Page
Part of larger #136 for all components.

### PR DESCRIPTION
**Pull request by droshev**
fixes #273

### CODE REVIEW

File: scss/layout/_page.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 @import "./../settings";
 @import "./../functions";
 @import "./../mixins";
+@import "./../core/root";
```
Comment (greg-a-smith) [inline scss/layout/_page.scss]: Why is this being imported?

File: test/templates/page/index.njk [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+
+{% block content %}
+    {% set example %}
+    <div style="border: 1px solid">
+        <article class="fd-page">
```
Comment (greg-a-smith) [inline test/templates/page/index.njk]: I know this has been spec'd this way, but `<article>` seems like the wrong tag to use for a "page".  Almost by definition, shouldn't this class be applied to a `<main>` tag?

From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main:
> The **HTML `<main>` element** represents the dominant content of the `<body>` of a document. The main content area consists of content that is directly related to or expands upon the central topic of a document, or the central functionality of an application.

File: components/page.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/layout/page";
+@import "../scss/icons/icon";
```
Comment (greg-a-smith) [inline components/page.scss]: Icons are being imported with no variable set and no font is being imported, but a variable is being set.  Something seems amiss. 😄 Seems like fonts are needed, but not icons.

File: scss/layout/_page.scss (hunk 1 above)
Comment (droshev) [reply scss/layout/_page.scss]: there are some css variables like `--fd-padding-ui` that are defined there

**Comment by greg-a-smith** [comment]
Created #283 as a follow-up story.

**Pull request merged**